src/test/java/nextstep/subway/unit/PathServiceTest.java

```diff
@@ -72,7 +72,7 @@ void getShortestPath() {
         lineRepository.save(lineShinbundang);
 
         //when
-        PathResponse shortestPath = pathService.getShortestPath(1L, 3L);
+        PathResponse shortestPath = pathService.getShortestPath(2L, 4L);
```

Comment:
매직 넘버를 활용하기 보다 테스트를 위해 생성해둔 객체를 활용해도 좋을 것 같아요 😃 
```suggestion
        PathResponse shortestPath = pathService.getShortestPath(gangnamStation.getId(), nambuTerminalStation.getId());
```